apache/datadog_checks/apache/apache.py

```diff
@@ -57,6 +58,9 @@ def check(self, instance):
 
         disable_ssl_validation = _is_affirmative(instance.get('disable_ssl_validation', False))
 
+        if disable_ssl_validation:
+            requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
```

Comment:
Given that we've imported `urllib3.exceptions` directly I think we could simply do `urllib3.disable_warnings(category=InsecureRequestWarning)`? https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings